core/contracts/financial-templates/implementation/Liquidatable.sol

```diff
@@ -171,26 +171,43 @@ contract Liquidatable is PricelessPositionManager {
      * @dev This method generates an ID that will uniquely identify liquidation
      * for the sponsor.
      * @param sponsor address to liquidate
-     * @param amountToLiquidate amount of liquidatedCollateral that liquidator wants to liquidate
+     * @param collateralPerToken abort the liquidation if the position's collateral per token exceeds this value
+     * @param maxTokensToLiquidate max number of tokes to liquidate
      * @return uuid of the newly created liquidation
      */
-    function createLiquidation(address sponsor, FixedPoint.Unsigned calldata amountToLiquidate)
-        external
-        fees()
-        onlyPreExpiration()
-        returns (uint uuid)
-    {
-        // Attempt to retrieve Position data for sponsor
+    function createLiquidation(
```

Comment:
WDYT about adding a TODO to limit how small liquidations can be (in the case where it's a partial)?

Reply:
+1 and also partial liquidations that are very close to 100% (for example, `tokensOutstanding- 1 wei`

Reply:
Added a TODO, both cases are problematic I think.